editoast/src/views/v2/timetable.rs

```diff
@@ -74,46 +68,33 @@ enum TimetableError {
     InfraNotFound { infra_id: i64 },
 }
 
-/// Creation form for a Timetable
-#[derive(Serialize, Deserialize, Derivative, ToSchema)]
-#[derivative(Default)]
-struct TimetableForm {
-    #[serde(default)]
-    pub electrical_profile_set_id: Option<i64>,
-}
-
-/// Creation form for a Timetable
+/// Creation result for a Timetable
 #[derive(Debug, Default, Serialize, Deserialize, Derivative, ToSchema)]
+#[cfg_attr(test, derive(PartialEq))]
 struct TimetableResult {
-    pub id: i64,
-    pub electrical_profile_set_id: Option<i64>,
+    pub timetable_id: i64,
```

Comment:
Completely inconsequential nit: any motivation for the `id` → `timetable_id` rename? Since the struct already has "Timetable" in the name `id` may be descriptive enough already?

Reply:
The reasoning is that the endpoint's response is pure json so "id" isn't extremely descriptive. You can deduce it's a timetable_id from the name of the endpoint, but we might as well be extra clear ?